### ISSUE TITLE
Homepage analysis

### ISSUE THREAD

**Issue by teleyinex** [Scifabric/pybossa#356]
Using [Steve Krug book Don't make me think](http://www.sensible.com/dmmt.html) I've been analyzing the home page of PyBossa. Steve Krug wrote a full section (chapter 7) about the home page in terms of building a good UX for your users.

Steve explains that a good home page is a web page that can answer the next 4 questions at a glance: 
- What _is_ this?
- What do they _have_ here?
- What can I _do_ here?
- Why I should be here --and not somewhere else?

He argues that if it is not clear to him what he is looking at in the first few seconds, interpreting everything else on the page is harder.

But if the user "gets it", it is much more likely that the user will interpret correctly everything he sees in the page.

From my point of view I like a lot his approach, and I run the "test" questions, and these are the results:
- What _is_ this?
  - Online assistance in performing tasks that require human cognition, knowledge or intelligence such as image classification, transcription, geocoding and more!
- What do they _have_ here?
  - Not _really_ clear, IMHO. The home page describes tasks, and there is the word application but not an explanation about it.
- What can I _do_ here?
  - Help advance research. Contributing. Again, IMHO the text should give a bit more of feedback.
- Why I should be here --and not somewhere else?
  - IMHO there is no a clear motivation or catchy sentence about this, except "Help advance research".

In summary, I think we need to:
- [ ] Explain what do we have here: web applications with tasks that need volunteers to get them solved.
- [ ] Improve the explanation about what I can do here: You can help to advance research, damage assessment, etc.
- [ ] Give a good reason why you should be here: because you can participate in real scientific projects, or create your own one easily.

@PyBossa  Comments?
